lib/handler/message_reaction_add.go

```diff
@@ -0,0 +1,16 @@
+package handler
+
+import (
+	"github.com/tpc3/DuckPolice/lib/config"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func MessageReactionAdd(session *discordgo.Session, reaction *discordgo.MessageReactionAdd) {
+	msg, _ := session.ChannelMessage(reaction.ChannelID, reaction.MessageID)
```

Comment:
Search in state cache first.
If not found, receive using session.

Reply:
OK